Gemfile

```diff
@@ -21,6 +21,7 @@ manageiq_plugin "manageiq-providers-ansible_tower" # can't move this down yet, b
 manageiq_plugin "manageiq-schema"
 
 # Unmodified gems
+gem "active_record-virtual_attributes", "~>0.1.0"
```

Comment:
Super minor, but the gem should probably be named `activerecord-virtual_attributes` for consistency (yeah I know it's wrong, but that's the way `activerecord` is named 😦).  I did something similar with `activerecord-id_regions`, and had to deal with that here: https://github.com/ManageIQ/activerecord-id_regions/tree/master/lib

Reply:
@kbrock ok, to verify, the gem name is `activerecord-virtual_attributes` but the directory structure is `lib/active_record/virtual_attributes/*`